### PR TITLE
updated German translations

### PR DESCRIPTION
**Pull request by plata**

### CODE REVIEW

File: i18n/de.po [hunk 1]
```diff
@@ -169,208 +95,224 @@ msgstr "Breeze Dunkel"
...
+#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/addrepository/RepositoryType.java:18
+msgid "Local repository"
+msgstr "Lokales Repository:"
 
-#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/WinePrefixContainerWineToolsTab.java:96
-msgid "Task manager"
-msgstr "Taskmanager"
+#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/addrepository/RepositoryType.java:28
+msgid "Git repository"
+msgstr "Git Repository:"
```
Comment (madoar) [inline i18n/de.po]: I don't think the `:` should be added to the repository titles.

Comment (plata) [inline i18n/de.po]: True. (Didn't even change this.)

File: i18n/de.po [hunk 2]
```diff
@@ -475,149 +455,173 @@ msgstr ""
...
+#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ViewApps.java:70
+msgid "Apps"
+msgstr "Anwendungen"
 
-#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/addrepository/AddRepositoryDialog.java:59
-msgid "Add a new Repository"
-msgstr "Repository hinzufügen:"
+#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ApplicationSidebar.java:123
+msgid "Testing"
+msgstr "in Testphase"
```
Comment (madoar) [inline i18n/de.po]: `I` not `i` I think

File: i18n/de.po [hunk 3]
```diff
@@ -475,149 +455,173 @@ msgstr ""
...
+msgid "Please wait while we are verifying the file ..."
+msgstr "Bitte warten Sie während wir die Datei überprüfen ..."
 
-#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/AboutPanel.java:87
-msgid "Build Timestamp:"
-msgstr "Zeitstempel des Builds:"
+#: phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java:133
+#, java-format
+msgid "This wizard will help you install {0} on your computer"
+msgstr ""
```
Comment (madoar) [inline i18n/de.po]: Why does this msgstr start with an empty string?

Comment (plata) [inline i18n/de.po]: No idea. It's not visible like this in poedit. I changed it anyways.

**Pull request merged**